MiniApp/Classes/Extensions/MiniApp+FileManager.swift

```diff
@@ -11,10 +11,21 @@ extension FileManager {
      * @return { URL } - URL path to the MiniApp.
      */
     class func getMiniAppDirectory(with appId: String, and versionId: String) -> URL {
+        return getMiniAppDirectory(with: appId).appendingPathComponent("\(versionId)/")
+    }
+
+    /*
+     * Provide the MiniApp directory by appending the system
+     * cache directory with AppID.
+     * @param { appId: String } - AppID of the MiniApp.
+     * @param { versionId: String } - VersionID of the MiniApp.
+     * @return { URL } - URL path to the MiniApp.
+     */
+    class func getMiniAppDirectory(with appId: String) -> URL {
```

Comment:
Maybe a different method name? I see two same methods returning different path

Reply:
I rename the other one to getMiniAppVersionDirectory which is more accurate